File: letter/ru/В.py
from common import *

expr = Eq(
    (x + S(1) / 2)
    * ((x + S(1) / 2) ** 2 + (2 * Abs(y) - 1) ** 2 - 1)
    * sqrt(
        S(x + S(1) / 2 + eps) /
        ((Abs(y) - 1 - eps) * (x - S(1) / 2 - eps))
    )
    ,
    0
)
size = 1

File: letter/ru/В.js
import { Letter } from '../../Letter.js';

const expr = '(x + 1/2) * ((x + 1/2)^2 + (2*abs(y) - 1)^2 - 1) * sqrt((x + 1/2 + 1/10)/((abs(y) - 1 - 1/10) * (x - 1/2 - 1/10)))';

export const letter = new Letter(expr, 1);
